metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -2937,4 +2940,45 @@ class MetalsLspService(
     }
   }
 
+  private def resetWorkspace(): Future[Unit] = {
+    def clearBloopDir(): Unit = {
+      val bloopDir = workspace.resolve(".bloop")
+      val directories = bloopDir.list.filter(_.isDirectory).toList
+
+      try {
+        directories.foreach(_.deleteRecursively())
+        val remainingDirs =
+          bloopDir.list.filter(f => f.exists && f.isDirectory).toList
+        if (remainingDirs.isEmpty) {
+          scribe.info("Deleted all the directories inside .bloop directory.")
+          tables.cleanAll()
+        } else {
+          val str = remainingDirs.mkString(", ")
+          scribe.error(
+            s"Couldn't delete all the directories in .bloop directory, remaining: $str"
+          )
+        }
+      } catch {
+        case e: Throwable =>
+          languageClient.showMessage(Messages.ResetWorkspaceFailed)
+          scribe.error(
+            "Couldn't delete all the directories inside .bloop: " +
+              e.getMessage()
+          )
```

Comment:
scribe.error can also accept `e` as a second argument. Can you check which version prints more information?